api/api.go

```diff
@@ -34,18 +34,24 @@ LICENSE
 package main
 
 import (
+	"github.com/ausocean/openfish/api/ds_client"
 	"github.com/ausocean/openfish/api/handlers"
 
+	"flag"
+	"fmt"
+
 	"github.com/gofiber/fiber/v2"
 )
 
-func main() {
-	app := fiber.New()
+// RegisterAPIRoutes registers all handler functions to their routes.
+func RegisterAPIRoutes(app *fiber.App) {
```

Comment:
This doesn't really need to be exported given that it's in the main package right ?